### PR TITLE
#151991927 Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by tosmak16**
#### What does this PR do?
It integrates Travis-CI with readMe badge
#### Description of Task to be completed?
**Integrate TravisCi with readme badge by doing the following:**
* Synchronize TravisCI with mont-blanch-cfc repo
* Create .travis.yml file in local repository 
* push to Github
* Copy build pass markdown from TravisCi
* Add markdown to readme file as readme badge
* push to Github
#### How should this be manually tested?
* Log into travisCI with Github account
* Get admin access to mont-blanch-cfc repo if need be 
* Sync Github account to TravisCI
* Click mont-blanch-cfc repo link on Travis CI
* Click branches
* Click chore/151991927/integrate-travisCI 
#### Any background context you want to provide?
none for now.
#### What are the relevant pivotal tracker stories?
#151991927
#### Screenshots
image

image

#### Questions:
none for now

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+MONGOHQ_URL=localhost
```
Comment (temilaj) [inline .env]: How is this file related to integrating Travis CI with the readme badge?

File: server.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+require('dotenv').config();
```
Comment (temilaj) [inline server.js]: same comment as above.

**Review by temilaj — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (tosmak16) [reply .env]: I have done necessary changes. Thanks.

File: server.js (hunk 2 above)
Comment (tosmak16) [reply server.js]: I have done necessary changes. Thanks.

**Pull request closed without merging**